.github/workflows/release.yml

```diff
@@ -1,113 +1,52 @@
-name: "Release (latest)"
+name: Release
 
 on:
-  workflow_dispatch:
-
-  workflow_call:
-    secrets:
-      TIGERBEETLE_NODE_PUBLISH_KEY:
-        required: true
-      NUGET_KEY:
-        required: true
-      MAVEN_GPG_SECRET_KEY:
-        required: true
-      MAVEN_CENTRAL_USERNAME:
-        required: true
-      MAVEN_CENTRAL_TOKEN:
-        required: true
-      MAVEN_GPG_SECRET_KEY_PASSWORD:
-        required: true
-      TIGERBEETLE_DOCS_DEPLOY_KEY:
-        required: true
-      TIGERBEETLE_GO_DEPLOY_KEY:
-        required: true
-      SLACK_TOKEN:
-        required: true
-    inputs:
-      version:
-        required: false
-        type: string
+  push:
+    branches:
+      - release
```

Comment:
Releases are triggered by pushes to the release branch. long term, we want something here to make sure at least two people sing off on a release, to protect from any single person's laptop being hacked.

Reply:
I feel we should definitely set it up as a protected branch on GitHub. I know I'll try accidentally push directly to it at least once :smile: 

Not sure if you've checked out the rules there but might be easy enough to require two person signoff etc too.